std_msgs/Header header

sensor_msgs/Image detections_image
ObjectDetection[] detections
